packages/tsutils/src/period/period.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+const DAY = 'DAY';
+const WEEK = 'WEEK';
+const MONTH = 'MONTH';
+const YEAR = 'YEAR';
+const QUARTER = 'QUARTER';
+
+/**
+ * Available period formats for aggregation server data
+ */
+export const PERIOD_TYPES = {
```

Comment:
This should be enum?

Reply:
Good point! I left this as is to make the migration easier and a bit safer.

If you don't mind, I can change it to an `enum` in a separate PR, after all the depending utils have moved under `tsutils` too